test/classify.ts

```diff
@@ -56,10 +56,10 @@ describe('The classify endpoint', () => {
     expect(response.body.classifications[0].prediction).to.equal('color'); // pink
     expect(response.body.classifications[1].prediction).to.equal('food'); // egg
     expect(response.body.classifications[2].prediction).to.equal('food'); // pasta
-  });
+  }).timeout(6000);
```

Comment:
what is this for?

Reply:
oh prior to this change these seem to always be timing out with the default timeout value (which i think is like 2s?) and they on average take 4-5s so I put it at 6s just in case, lmk if that makes sense!